File: ip-messaging/accessmanager/initialization/initialization.m
self.accessManager = [TwilioAccessManager accessManagerWithToken:token
                                                        delegate:self];
[TwilioChatClient chatClientWithToken:accessManager.currentToken
                           properties:properties
                             delegate:delegate
                           completion:(TCHResult *result, TwilioChatClient *client) {
  if ([result isSuccessful]) {
    self.client = client;
    __weak typeof(client) weakClient = client;
    [accessManager registerClient:client forUpdates:^(NSString *updatedToken) {
        [weakClient updateToken:updatedToken completion:^(TCHResult *result) {
          if (![result isSuccessful]) {
            // warn the user the update didn't succeed
          }
        }];
    }];
  }
 }];

File: ip-messaging/accessmanager/initialization/initialization.swift
self.accessManager = TwilioAccessManager.init(token: token, delegate: self)
TwilioChatClient.init(token: accessManager.token, properties: properties, delegate: delegate) { (result, client) in
  if (result.isSuccessful()) {
    self.client = client
    self.accessManager?.registerClient(client, forUpdates: { [weak client = self.client] (token) in
        client?.updateToken(token) { (result) in
          if (!result.isSuccessful()) {
            // warn the user the update didn't succeed
          }
        }
      })
  }
}

